typescript/src/actions/evm/transfer/transferWithAccount.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  createPublicClient,
+  http,
+  erc20Abi,
+  parseUnits,
+  Hex,
+  encodeFunctionData,
+  Address,
+  TransactionReceipt,
+  WaitForTransactionReceiptTimeoutError,
+} from "viem";
+
+import { TransferResult, TransferOptions } from "./types.js";
+import { getErc20Address, mapNetworkToChain } from "./utils.js";
+import { EvmAccount } from "../../../accounts/types.js";
+import { CdpOpenApiClientType } from "../../../openapi-client/index.js";
+import { serializeEIP1559Transaction } from "../../../utils/serializeTransaction.js";
+
+/**
+ * Transfer an amount of a token from an account to another account.
+ *
+ * @param apiClient - The client to use to send the transaction.
+ * @param from - The account to send the transaction from.
+ * @param transferArgs - The options for the transfer.
+ *
+ * @returns The result of the transfer.
+ */
+export async function transfer(
```

Comment:
This function is more or less duplicated with a counterpart in `transferWithSmartAccount`. Opted to keep the code dumb and simple for now, but can see us wanting to reduce some code duplication between these

Reply:
Added a transaction execution strategy abstraction to handle this!